code/modules/uplink/uplink_items_fulp.dm

```diff
@@ -89,4 +89,23 @@
 			in electronic devices, subverts intended functions, and easily breaks security mechanisms.  \
 			This is a cheap knockoff Space-China budget version that holds 2 charges, and regains 1 charge every 30 seconds."
 	item = /obj/item/card/emag/budget
-	cost = 6
\ No newline at end of file
+	cost = 6
+
+/datum/uplink_item/role_restricted/shelter_traitor
+	name = "Tacticool Bluespace Shelter"
+	desc = "A fairly simple concept of bluespace capsule redesigned by Syndicate. \
```

Comment:
make sure to mention the advanced camera console that's disguised as a pod computer in this description, since that's the main draw of this item